data/mods/ssb/moves.ts

```diff
@@ -1083,6 +1083,58 @@ export const BattleMovedex: {[k: string]: ModdedMoveData} = {
 		type: "Bird",
 	},
 
+	// LittEleven
+	nexthunt: {
+		accuracy: true,
+		basePower: 0,
+		category: "Status",
+		desc: "If this Pokemon does not take damage this turn, it switches out to another Pokemon in the party and gives it the Download boost. Fails otherwise.",
+		shortDesc: "If not damaged this turn, switches out and gives next Pokemon Download boost; fails otherwise.",
+		name: "/nexthunt",
+		pp: 10,
+		priority: 0,
+		flags: {protect: 1, mirror: 1},
+		sideCondition: "nexthunt",
+		onTryMove() {
+			this.attrLastMove('[still]');
+		},
+		onPrepareHit(target, source) {
+			this.add('-anim', source, 'Teleport', source);
+		},
+		onTryHit(target, source, move) {
+			if (!move.selfSwitch) return;
+		},
+		effect: {
+			duration: 1,
+			onHit(pokemon, source, move) {
```

Comment:
So, this is going to activate regardless because its not looking for when damage is taken before the turn starts

Reply:
I think the best way to do this is to move all this into a different side condition under tracker in statuses.js, then make a clone of the focuspunch volatile and apply it here. Or vise versa idc